internal/pkg/db/mongo/models/action.go

```diff
@@ -14,42 +14,8 @@
 
 package models
 
-import (
-	contract "github.com/edgexfoundry/edgex-go/pkg/models"
-	"github.com/pkg/errors"
-)
-
 type Action struct {
```

Comment:
I would probably do away with this type since everything about it is managed in the Get/Put types. It's literally got zero responsibilities and exists only for this reason:
```
type Get struct {
	Action `bson:",inline"`
}
```
which is an artifact of the way the original contract was written -- kind of like with `BaseObject`

Reply:
Yep.  Removed.